### DISCUSSION TITLE
p/7ebabda6f3a44a9b83cc5c5109b126e2/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#51] [Announcements]
# p/7ebabda6f3a44a9b83cc5c5109b126e2/

这周主要是报团前往了新疆，由于新疆西藏等地都是地域辽阔交通不便，所以单独为其规划了行程。考虑到是第一次来到新疆，所以并未选择自由行而是报了个团。  乌鲁木齐第一站飞到了乌鲁木齐，中午抵达后就在市区内逛了逛。 首先去了红山公园，这个公园的水非常脏，是天津之后第二次见到水面上漂浮着死鱼的水，非常糟糕。 在山上似乎可以看到。。。双子塔？  下山途中在一处廖无人烟之处看到了墙画，上面涂满了不止从何而来的乱

https://blog.rxliuli.com/p/7ebabda6f3a44a9b83cc5c5109b126e2/

**Comment by miaopass-future** [comment]
已经完成留学了么

**Reply by rxliuli** [reply]
不，已经滚回国内了，适应不了那边

**Reply by miaopass-future** [reply]
这样么，太可惜了。感觉如果不完成的话会损失很多呢，时间、金钱之类的

**Reply by rxliuli** [reply]
是这样，但出去看看才知道是否合适，知道自己想要什么。无论如何，半年倒也还好。目前正在全国旅行中，想要走走之前几乎完全忽视的国内风景

**Reply by rxliuli** [reply]
甚至到目前为止，在日本走过的地方也多过国内。
![Screenshot_20240514-163839_Swarm](https://github.com/rxliuli/blog/assets/24560368/2dbf10f7-b5e7-4500-b190-d05cfd459b34)
